<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>select_color_page</class>
 <widget class="QWidget" name="select_color_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>5</y>
     <width>250</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(19,20,20);
border-radius:4px;</string>
   </property>
   <widget class="QPushButton" name="blue_btn">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>10</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgba(76,119,231,1);
border-radius:2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pink_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgba(250,108,99,1);
border-radius:2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="dark_green_btn">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>10</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgba(15,161,90,1);
border-radius:2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="orang_btn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgba(255,151,47,1);
border-radius:2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Violet_btn">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>10</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgba(186,123,216,1);
border-radius:2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Golden_btn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgba(248,209,93,1);
border-radius:2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="light_blue_btn">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>10</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgba(42,162,217,1);
border-radius:2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="white_btn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgba(236,238,242,1);
border-radius:2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="yellow_btn">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>10</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgba(250,243,175,1);
border-radius:2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="light_green_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgba(110,207,67,1);
border-radius:2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
